tap_github/streams.py

```diff
@@ -158,6 +157,40 @@ def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
     ).to_dict()
 
 
+class ReadmeStream(GitHubStream):
+    name = "readme"
+    path = "/repos/{org}/{repo}/readme"
+    primary_keys = ["url"]
+    parent_stream_type = RepositoryStream
+    ignore_parent_replication_key = False
+    state_partitioning_keys = ["repo", "org"]
+
+    def parse_response(self, response: requests.Response) -> Iterable[dict]:
+        return [response.json()]
+
+    schema = th.PropertiesList(
+        th.Property("type", th.StringType),
```

Comment:
Can we add the parent keys `repo` and `org` here? If you declare them in the properties list, they should get populated automatically from parent context.